lib/routes/utils/oauth-mint.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
That thing that this module is doing is weird enough that I think it could benefit from a short module-level comment explaining what's going on.